### ISSUE TITLE
Cannot visualize (flux diagram) Liquid Phase RMG-Py File

### ISSUE THREAD

**Issue by DetlevCM** [ReactionMechanismGenerator/RMG-Py#588]
For a liquid phase rmg py scheme, I am presented with an error by the visualisation tool.
This is the error for all intents and purposes (don't think the Django stuff matters). This occurs both locally and on the website itself

```
Environment:


Request Method: POST
Request URL: http://rmg.mit.edu/simulate/flux

Django Version: 1.8.5
Python Version: 2.7.6
Installed Applications:
('django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.sites',
 'django.contrib.messages',
 'django.contrib.admin',
 'rmgweb.main',
 'rmgweb.main.templatetags',
 'rmgweb.database',
 'rmgweb.pdep',
 'rmgweb.rmg')
Installed Middleware:
('django.middleware.common.CommonMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware')


Traceback:
File "/opt/anaconda/envs/rmg/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  132.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/var/www/RMG-website/rmgweb/rmg/views.py" in generateFlux
  199.             createFluxDiagram(flux.path, input, chemkin, dict, java, settings, chemkinOutput)
File "/var/www/RMG-Py/rmgpy/tools/fluxdiagram.py" in createFluxDiagram
  452.     rmg = loadRMGJob(inputFile, chemkinFile, speciesDict, java)
File "/var/www/RMG-Py/rmgpy/tools/loader.py" in loadRMGJob
  49.         rmg = loadRMGPyJob(inputFile, chemkinFile, speciesDict)
File "/var/www/RMG-Py/rmgpy/tools/loader.py" in loadRMGPyJob
  85.         reactionSystem.initialMoleFractions = dict([(speciesDict[spec], frac) for spec, frac in reactionSystem.initialMoleFractions.iteritems()])

Exception Type: AttributeError at /simulate/flux
Exception Value: 'rmgpy.solver.liquid.LiquidReactor' object has no attribute 'initialMoleFractions'

```

**Comment by bslakman** [comment]
For a liquid-phase reactor, we don't have `initialMoleFractions`, instead we have `initialConcentrations`. I am not sure how the flux diagram visualization works exactly, but we should either load the initial concentrations instead, or perhaps we can convert them to mole fractions if required using the solvent density?

**Comment by DetlevCM** [comment]
Unless I am missing something, you wouldn't need any density to work out mol fractions from concentrations, just sum and normalize.

IF the numbers can be obtained too, concentrations would make more sense than mol fractions as then these can be plotted and used more conveniently, but for just the flux diagram either would work.

**Comment by cgrambow** [comment]
Should be fixed with #1225.
